ci/run_tests.sh

```diff
@@ -5,19 +5,24 @@ set -euxo pipefail # https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_
 
 if [ "$emscripten_supported" == "0" ]; then
   # TODO - Emscripten builds are broken on rustc > 1.39.0
-  cargo web test --features web_test --target asmjs-unknown-emscripten
-  cargo web test --features web_test --target wasm32-unknown-emscripten
+  cargo web test --features web_test,std_web --target asmjs-unknown-emscripten
```

Comment:
Did you intentionally change the permissions on this script file? `100755 → 100644`

Reply:
No, this is just some windows bullshittery. Will try to revert this.

Reply:
Fixed.